examples/dcomexec.py

```diff
@@ -554,7 +567,8 @@ def load_smbclient_auth_file(path):
 
     parser.add_argument('command', nargs='*', default = ' ', help='command to execute at the target. If empty it will '
                                                                   'launch a semi-interactive shell')
-
+    parser.add_argument('-silentcommand', action='store_true', default = False, help='does not execute a command '
+                        'proccessor to run given command (cannot run dir/cd/etc.)')
```

Comment:
We might add here that there's no output generated. Also we might want to unify help messages.